### PR TITLE
discord.py 1.7.1

### PR DESCRIPTION
**Pull request by LucasCoderT**

### CODE REVIEW

File: docsets/discord.py/docset.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
         {
             "version": "1.6.0",
             "archive": "versions/1.6.0/discord.py.tgz",
+        },
+        {
+            "version": "1.7.1",
```
Comment (Kapeli) [inline docsets/discord.py/docset.json:17]: Newest versions should be added at the top, not bottom

**Comment by Kapeli** [comment]
Please don't include the unarchived docset in your pull request, only include an archive.

File: docsets/discord.py/docset.json [hunk 2]
```diff
@@ -12,6 +12,10 @@
...
+            "archive": "versions/1.7.1/discord.py.tgz",
```
Comment (Kapeli) [inline docsets/discord.py/docset.json:18]: This file doesn't exist

File: docsets/discord.py/docset.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "discord.py",
-    "version": "1.6.0",
+    "version": "1.7.1",
     "archive": "discord.py.tgz",
```
Comment (Kapeli) [inline docsets/discord.py/docset.json:4]: You need to update the docset archive at this path as well.

**Comment by LucasCoderT** [comment]
Apologies, should be fixed now

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**